source/languages/en/riakcs/cookbooks/configuration/Configuring-Riak.md

```diff
@@ -157,32 +206,45 @@ If the response is `pong`, then Riak is running; if the response is
 If the node has not started properly, look at the `erlang.log.1` in the
 `/log` directory of the node to see if the problem can be identified.
 One common error is `invalid_storage_backend`, which indicates that the
-path to the Riak CS library in `app.config` is incorrect (or that Riak
-CS is not installed on the server). In spite of this error, make sure
-that you do not change the backend from `riak_cs_kv_multi_backend` to
+path to the Riak CS library in `advanced.config` or in `app.config` is incorrect
+(or that Riak CS is not installed on the server). In spite of this error, make
+sure that you do not change the backend from `riak_cs_kv_multi_backend` to
 `riak_kv_multi_backend`.
 
 ## Setting Up Riak to Use Protocol Buffers
 
-The Riak [[Protocol Buffers|PBC API]] settings reside in the Riak
-`app.config` file, which is located in the `/etc/riak` folder. The
-settings appear in the `riak_api` section of the file. The default host
-is 127.0.0.1 while the default port is 8087. You will need to change
-this if you plan on running Riak and Riak CS in a non-local environment.
-Replace `127.0.0.1` with the IP address of the Riak node and `8087` with
-the port:
+The Riak [[Protocol Buffers|PBC API]] settings reside in the Riak `riak.conf`,
+or in the `riak_api` section of the the old-style `advanced.config` or
+`app.config` files, which is located in the `/etc/riak/` folder. The default host
+is `127.0.0.1` and the default port is `8087`. You will need to change this if
+you plan on running Riak and Riak CS in a non-local environment. Replace
+`127.0.0.1` with the IP address of the Riak node and `8087` with the appropriate
+port:
+
+```riakconf
+listener.protobuf.internal = 10.0.2.10:10001
+```
+
+```advancedconfig
+{riak_api, [
+    %% Other configs
+    {pb, ["10.0.2.10", 10001]},
+    %% Other configs
+]}
+```
 
 ```appconfig
 {riak_api, [
     %% Other configs
-    {pb, ["<IP for your node>", <port>]},
+    {pb, ["10.0.2.10", 10001]},
     %% Other configs
 ]}
 ```
 
-**Note**: The `pb` values in the Riak `app.config` file must match the
-values for `riak_ip` and `riak_pb_port` in the Riak CS and Stanchion
-`app.config` files.
+**Note**: The `listener.protobuf.internal` values in the Riak `riak.conf` (or
+the `pb` value in `advanced.config`/`app.config`) file must match the values for
+`riak_host`in the Riak CS and Stanchion `riak-cs.config` (or `riak_ip` and
+`riak_pb_port` in `advanced.config`/`app.config`) files.
```

Comment:
I should have apologised that here old behaviour is not preserved. In `advanced.config`, the item is `{riak_host, {localhost, 8087}},` .